frontend/package.json

```diff
@@ -15,8 +15,10 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-icons": "^4.7.1",
+    "react-lite-youtube-embed": "^2.3.52",
     "react-router-dom": "^6.4.4",
     "react-youtube": "^10.1.0",
```

Comment:
remove useless dependency

Reply:
This is the good dependecy

Reply:
If you use `react-lite-youtube-embed`, you don't need `react-youtube` anymore, right ? Gerald is right, you could remove the old one